match/match.py

```diff
@@ -20,7 +20,7 @@ class Match(commands.Cog):
     MATCHES_KEY = "Matches"
 
     def __init__(self, bot):
-        self.bot = bot
+        self.bot= bot
```

Comment:
Why did you delete a space? What did that space ever do to you? :(

Reply:
IT WAS A MISTAKE, I DIDN'T MEAN TO 😢